engine-spring/core/src/main/java/org/operaton/bpm/engine/spring/components/config/xml/StateHandlerAnnotationBeanFactoryPostProcessor.java

```diff
@@ -40,7 +40,7 @@
 public class StateHandlerAnnotationBeanFactoryPostProcessor implements BeanFactoryPostProcessor {
 
 	private ProcessEngine processEngine ;
-	private Logger log = Logger.getLogger(getClass().getName());
+  private final Logger log = Logger.getLogger(getClass().getName());
```

Comment:
This change caused an uneven layout in the class itself

Reply:
It looks so, but it has just replaced a tab by spaces, actually cleaned up here